setup.py

```diff
@@ -76,7 +76,7 @@
         'collatex==2.1.2',
         'hdbscan',
         'lxml',
-        'numpy>=1.14.5',
+        'numpy==1.15.0',
```

Comment:
I had to bump this to `1.16.0` to get it working with db scan - https://github.com/scikit-learn-contrib/hdbscan/issues/272

Reply:
You can safely change this to `numpy==1.16.2`, I have tested it in a clean (conda) environment on my mac and run through the full workshop example without issue.  `skimage` and `hdbscan` both work as expected numpy's latest patch.

Reply:
```suggestion
        'numpy==1.16.2,
```